cf_cli_java_plugin.go

```diff
@@ -168,10 +177,23 @@ func (c *JavaPlugin) execute(commandExecutor cmd.CommandExecutor, uuidGenerator
 	}
 
 	var remoteCommandTokens = []string{JavaDetectionCommand}
-
+	remoteFileFullPath := ""
+	localFileFullPath:= ""
 	switch command {
 	case heapDumpCommand:
-		heapdumpFileName := "/tmp/heapdump-" + uuidGenerator.Generate() + ".hprof"
+
+		valid, _ :=utils.CheckRequiredTools(applicationName)
+		if !valid{
+			return "", nil
+		}
+		
+		fspath, _ := utils.GetAvailablePath(applicationName, remoteDir)
+		//fmt.Println(fspath)
+		dumpFile := applicationName + "-heapdump-" + uuidGenerator.Generate() + ".hprof"
+		heapdumpFileName := fspath + "/" + dumpFile
+		remoteFileFullPath = fspath + "/" + dumpFile
```

Comment:
Why is `heapdumpFileName` and `remoteFileFullPath` the same?

Reply:
removed duplicate